website/_config.yml

```diff
@@ -0,0 +1,19 @@
+title: Krayon
+description: Cross platform (Android, iOS & JavaScript) canvas library.
+
+plugins:
+  - jekyll-remote-theme
+  - jekyll-spaceship
+
+remote_theme: jekyll/minima
+
+kramdown:
+  parse_block_html: true
```

Comment:
This option was needed otherwise codeblocks within `<details>` blocks would not render correctly.